lib/MusicBrainz/Server/Edit/Alias/Edit.pm

```diff
@@ -172,6 +174,8 @@ sub initialize
         $opts{sort_name} = $opts{name} if non_empty($opts{name});
```

Comment:
Would it make sense to move this chunk of code to `enforce_dependencies` too? Edit::Alias::Add has similar logic, except it doesn't `delete $opts{sort_name}` or check `if non_empty($opts{name})`.